api_docs/changelog.md

```diff
@@ -1292,6 +1292,8 @@ field with an integer field `invite_to_realm_policy`.
 * [`GET /events`](/api/get-events): Added new event type `muted_users`
   which will be sent to a user when the set of users muted by them has
   changed.
+* [`POST /register`](/api/register-queue): Added a new `muted_users`
+  field, which identifies the set of other users they have muted.
```

Comment:
I think the "they" here doesn't have a clear referent. Could say instead "the current user":
```suggestion
* [`POST /register`](/api/register-queue): Added a new `muted_users`
  field, which identifies the set of other users the current user has muted.
```

like we do in a few other places on this page.

Reply:
Thanks for taking a look, I agree this reads better :+1:
Pushed an update.